commands.json

```
@@ -397,6 +397,72 @@
     "since": "6.2.0",
     "group": "list"
   },
+  "LMPOP": {
+    "summary": "Pop elements from a list",
+    "complexity": "O(N+M) where N is the number of provided keys and M is the number of elements returned.",
+    "arguments": [
+      {
+        "name": "numkeys",
+        "type": "integer"
+      },
+      {
+        "name": "key",
+        "type": "key",
+        "optional": true,
+        "multiple": true
+      },
+      {
+        "name": "where",
+        "type": "enum",
+        "enum": [
+          "LEFT",
+          "RIGHT"
+        ]
+      },
+      {
+        "name": "count",
```

I think we need to explicitly add the `COUNT` keyword before the count, so this should be
```suggestion
        "command": "COUNT",
        "name": "count",
```